### PR TITLE
Gml 1543 sync milvus tg backup handling

### PR DESCRIPTION
**Pull request by ahmed-tg**
- Added deleted vertex syncing to Milvus
- Created accompanying GSQL query
- Added query to the Milvus embedding store for more flexibility to get output fields

### CODE REVIEW

File: common/embeddings/milvus_embedding_store.py [hunk 1]
```diff
@@ -490,5 +490,31 @@ def list_registered_documents(
...
+        """Get output fields with expression
+
+        Args:
+            expr: Expression - E.g: "pk > 0"
+
+        Returns:
+            List of output fields' contents
+        """
+
+        from pymilvus import MilvusException
```
Comment (RobRossmiller-TG) [inline common/embeddings/milvus_embedding_store.py:503]: We can move this to the top so we don't have to import on every call

File: eventual-consistency-service/app/eventual_consistency_checker.py [hunk 2]
```diff
@@ -244,14 +249,50 @@ def fetch_and_process_vertex(self):
...
+            else:
+                time.sleep(self.interval_seconds)
+
+            
+    def initialize_cleanup(self):
+        LogWriter.info(
+            f"Eventual Consistency Check running cleanup for graphname {self.graphname} "
+        )
+        self.is_initialized = True
+        while True and self.run_forever:
```
Comment (RobRossmiller-TG) [inline eventual-consistency-service/app/eventual_consistency_checker.py:289]: Can just be while `self.run_forever`

File: eventual-consistency-service/app/eventual_consistency_checker.py [hunk 3]
```diff
@@ -244,14 +249,50 @@ def fetch_and_process_vertex(self):
...
+            self.verify_and_cleanup_embeddings()
+
+            if not self.run_forever:
```
Comment (RobRossmiller-TG) [inline eventual-consistency-service/app/eventual_consistency_checker.py:292]: if run_forever is false, the loop will never enter

**Review by RobRossmiller-TG — CHANGES_REQUESTED** [review]
A few small things, otherwise LGTM

**Review by RobRossmiller-TG — APPROVED** [review]

File: eventual-consistency-service/app/eventual_consistency_checker.py (hunk 2 above)
Comment (ahmed-tg) [reply eventual-consistency-service/app/eventual_consistency_checker.py]: oops forgot to move this, good catch

**Review by JextraInc — APPROVED** [review]

**Pull request merged**